config/helm/chart/default/templates/Common/csi/clusterrole-csi.yaml

```diff
@@ -62,7 +62,7 @@ rules:
       - get
       - list
       - watch
-  {{- if (eq (include "dynatrace-operator.openshiftOrOlm" .) "true") }}
+  {{- if (eq (include "dynatrace-operator.platform" .) "openshift") }}
```

Comment:
Sidenote: IMO, we should "rework" `openshiftOrOlm`, so that it does not have the `(include "dynatrace-operator.partial" .) "false")` part as the name in no way describes that behaviour, and that is the reason this bug snuck in.

```
{{- define "dynatrace-operator.openshiftOrOlm" -}}
{{- if and (or (eq (include "dynatrace-operator.platform" .) "openshift") (.Values.olm)) (eq (include "dynatrace-operator.partial" .) "false") -}}
    {{ default "true" }}
{{- end -}}
{{- end -}}

```

Reply:
will create a follow up for that :)